components/prism-nginx.js

```diff
@@ -0,0 +1,18 @@
+});
+
+Prism.languages.insertBefore('nginx', 'keyword', {
+        variable:/(\$[a-zA-Z_]+)/ig,
```

Comment:
You don't need the parentheses around the whole pattern. Also, you're using the `i` flag so you can simplify the character class a bit. Finally, there is no need for the `g` flag in Prism (it is not used).